test/ng/httpBackendSpec.js

```diff
@@ -405,6 +413,18 @@ describe('$httpBackend', function() {
     });
 
 
+    it('should throw error if the url is not a trusted resource', function() {
+      expect(function() {
+        $backend('JSONP', 'http://example.org/path?cb=JSON_CALLBACK', null, callback);
+      }).toThrowMinErr('$sce', 'insecurl');
+    });
+
+    it('should not throw error if the url is an explicitly trusted resource', function() {
+      expect(function() {
+        $backend('JSONP', $sce.trustAsResourceUrl('http://example.org/path?cb=JSON_CALLBACK'), null, callback);
+      }).not.toThrowMinErr('$sce', 'insecurl');
```

Comment:
I think generic `.not.toThrow()` expectations are preferrable, because a slight change in the error name (or a typo) render the test useless.